parachain-template/node/src/command.rs

```diff
@@ -181,25 +184,20 @@ pub fn run() -> Result<()> {
 			let runner = cli.create_runner(cmd)?;
 			// Switch on the concrete benchmark sub-command-
 			match cmd {
-				BenchmarkCmd::Pallet(cmd) =>
-					if cfg!(feature = "runtime-benchmarks") {
-						runner.sync_run(|config| cmd.run::<Block, ()>(config))
-					} else {
-						Err("Benchmarking wasn't enabled when building the node. \
+				#[cfg(not(feature = "runtime-benchmarks"))]
+				BenchmarkCmd::Pallet(_) => Err("Benchmarking wasn't enabled when building the node. \
 					You can enable it with `--features runtime-benchmarks`."
-							.into())
```

Comment:
Also here the Question; why not use the `cfg!` to make it look more compact instead of the `cfg(not`...?

Reply:
the only reason is that it is done with attributes in all of the other places.  So, just didn't want to mix multiple approaches.